controllers/component_build_controller_pac.go

```diff
@@ -148,6 +167,156 @@ func (r *ComponentBuildReconciler) ProvisionPaCForComponent(ctx context.Context,
 	return mrUrl, nil
 }
 
+func getHttpClient() *http.Client {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: gp.IsInsecureSSL()},
```

Comment:
it might look like git provider configuration but it is about the same pac webhook URL

https://github.com/redhat-appstudio/build-service/blob/main/controllers/component_build_controller_pac.go#L118
https://github.com/redhat-appstudio/build-service/blob/main/controllers/component_build_controller_pac.go#L129
https://github.com/redhat-appstudio/build-service/blob/main/controllers/component_build_controller_pac.go#L602
https://github.com/redhat-appstudio/build-service/blob/main/pkg/git/github/github_client.go#L219
https://github.com/redhat-appstudio/build-service/blob/main/pkg/git/github/github_client.go#L262

Reply:
Yes, you are right